tests/integration/components/polaris-button-group-test.js

```diff
@@ -57,6 +54,40 @@ test('it renders the correct HTML with segmented attribute', function(assert) {
   );
 });
 
+test('it renders the correct HTML with fullWidth attribute', function(assert) {
+  this.set('fullWidth', true);
+  this.render(hbs`{{polaris-button-group fullWidth=fullWidth}}`);
+
+  const buttonGroup = find(buttonGroupSelector);
+  assert.ok(
```

Comment:
why not something simpler
`assert.dom(buttonGroupSelector).hasClass('Polaris-ButtonGroup--fullWidth', 'fullWidth=false - does not add the fullWidth class');`

Reply:
Doh! Forgot to codemod this file..